src/class-fields/eval-err-contains-superproperty.case

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: `super().x` in StatementList of eval
+info: |
+  Additional Early Error Rules for Eval Inside Initializer
+  These static semantics are applied by PerformEval when a direct eval call occurs inside a class field initializer.
+  ScriptBody : StatementList
+
+    ...
+    The remaining eval rules apply as outside a constructor, inside a method, and inside a function.
+
+  Additional Early Error Rules for Eval Outside Methods
+  These static semantics are applied by PerformEval when a direct eval call occurs outside of a MethodDefinition.
+  ScriptBody:StatementList
+
+    It is a Syntax Error if StatementList Contains SuperProperty.
+features: [class-fields]
+template: initializer-eval-super
+---*/
+
+//- initializer
+super().x
```

Comment:
I believe this should be `super.x` to match a SuperProperty:

```
SuperProperty:
  super [ Expression ]
  super . IdentifierName
```

And this case may also be duplicated with `super['x']`